### ISSUE TITLE
OCommandSQLParsingException - Batch usage of $0 in traverse

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#5221]
Orientdb 2.1.4
Studio and Orientjs

Batch
```
LET $0 = select from Node  WHERE Mid = '562d6e990e59f2f43e9a70a2'  and BinPoints >= 30
LET $1 = select count(*) from ( traverse out() from $0 while Active = true and BinPoints >= 30 )
```

com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #0: Error parsing query: select count(*) from ( traverse out() from $0 while Active = true and BinPoints >= 30 ) Encountered " <FROM> "from "" at line 1, column 17. Was expecting one of: <EOF> <TIMEOUT> ... <AS> ... <FETCHPLAN> ... <LOCK> ... <LET> ... <NOCACHE> ... <PARALLEL> ... <UNWIND> ... ";" ... "," ... <AS> ... "," ...

**Comment by luigidellaquila** [comment]
Hi @StarpTech 

I just pushed a fix in branch 2.1.x, now I'm porting it to develop

**Comment by luigidellaquila** [comment]
pushed in develop branch. 
I'm closing this issue, please reopen in case I missed something

Thanks

Luigi
